agent/taskresource/cgroup/cgroup.go

```diff
@@ -284,11 +284,12 @@ func (cgroup *CgroupResource) Cleanup() error {
 
 // cgroupResourceJSON duplicates CgroupResource fields, only for marshalling and unmarshalling purposes
 type cgroupResourceJSON struct {
-	CgroupRoot      string        `json:"CgroupRoot"`
-	CgroupMountPath string        `json:"CgroupMountPath"`
-	CreatedAt       time.Time     `json:",omitempty"`
-	DesiredStatus   *CgroupStatus `json:"DesiredStatus"`
-	KnownStatus     *CgroupStatus `json:"KnownStatus"`
+	CgroupRoot      string               `json:"CgroupRoot"`
```

Comment:
There's a discrepancy wrt JSON labels between this and https://github.com/aws/amazon-ecs-agent/pull/1406/files#diff-321265a19fa4fe3d5d21afce0e0c481dR289. There, labels begin with lower case letters. Here, they begin with upper letters. We should stick with one format.